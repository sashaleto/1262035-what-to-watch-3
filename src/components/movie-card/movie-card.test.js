import React from "react";
import renderer from "react-test-renderer";
import MovieCard from './movie-card';

const film = {
  title: `The Great Gatsby`,
};

it(`Render MovieCard`, () => {
  const tree = renderer
        .create(<MovieCard
          film={film} onFilmTitleClick={() => {}} onMovieCardHover={() => {}}
        />)
        .toJSON();

  expect(tree).toMatchSnapshot();
});
