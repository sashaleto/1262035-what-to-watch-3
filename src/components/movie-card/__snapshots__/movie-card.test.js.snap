// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="The Great Gatsby"
      height="175"
      src="img/the-great-gatsby.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      The Great Gatsby
    </a>
  </h3>
</article>
`;
